<!-- Launch file for Ground Filter -->
<launch>
	<arg name="subscribe_topic" default="/velodyne_points" />
	<arg name="remove_ground" default="true" />
	<arg name="points_distance" default="0.2" />
	<arg name="angle_threshold" default="0.35" />
	<arg name="voxel_size_x" default="0.065" />
	<arg name="voxel_size_y" default="0.065" />
	<arg name="voxel_size_z" default="0.065" />
	
	<!-- Run ground_filter -->
	<node pkg="ground_filter" type="ground_filter_node" name="ground_filter_node">
		<param name="subscribe_topic" value="$(arg subscribe_topic)" />
		<param name="remove_ground" value="$(arg remove_ground)" />
		<param name="points_distance" value="$(arg points_distance)" />
		<param name="angle_threshold" value="$(arg angle_threshold)" />
		<param name="voxel_size_x" value="$(arg voxel_size_x)" />
		<param name="voxel_size_y" value="$(arg voxel_size_y)" />
		<param name="voxel_size_z" value="$(arg voxel_size_z)" />
	</node>
</launch>
